File: src/main/java/fi/aalto/cs/apluscourses/intellij/notifications/ExerciseNotSelectedNotification.java
package fi.aalto.cs.apluscourses.intellij.notifications;

import com.intellij.notification.Notification;
import com.intellij.notification.NotificationType;
import fi.aalto.cs.apluscourses.intellij.services.PluginSettings;

import static fi.aalto.cs.apluscourses.utils.PluginResourceBundle.getText;

public class ExerciseNotSelectedNotification extends Notification {

  /**
   * Constructs a notification that notifies the user that no exercise is selected. This should be
   * shown when the user uses the exercise submission button, but no exercise is selected.
   */
  public ExerciseNotSelectedNotification() {
    super(
        PluginSettings.A_PLUS,
        getText("notification.ExerciseNotSelectedNotification.title"),
        getText("notification.ExerciseNotSelectedNotification.content"),
        NotificationType.INFORMATION);
  }

}

File: src/main/kotlin/fi/aalto/cs/apluscourses/notifications/ExerciseNotSelectedNotification.kt
package fi.aalto.cs.apluscourses.notifications

import com.intellij.notification.Notification
import com.intellij.notification.NotificationType
import fi.aalto.cs.apluscourses.MyBundle
import fi.aalto.cs.apluscourses.services.PluginSettings

class ExerciseNotSelectedNotification
/**
 * Constructs a notification that notifies the user that no exercise is selected. This should be
 * shown when the user uses the exercise submission button, but no exercise is selected.
 */
    : Notification(
    PluginSettings.A_PLUS,
    MyBundle.message("notification.ExerciseNotSelectedNotification.title"),
    MyBundle.message("notification.ExerciseNotSelectedNotification.content"),
    NotificationType.INFORMATION
)
